x-pack/plugins/fleet/public/applications/fleet/sections/settings/index.tsx

```diff
@@ -123,7 +123,11 @@ export const SettingsApp = withConfirmModalProvider(() => {
         </Route>
         <Route path={FLEET_ROUTING_PATHS.settings_create_outputs}>
           <EuiPortal>
-            <EditOutputFlyout proxies={proxies.data.items} onClose={onCloseCallback} />
+            <EditOutputFlyout
+              proxies={proxies.data.items}
+              onClose={onCloseCallback}
+              defaultOuput={outputs.data?.items.find((o) => o.id === 'fleet-default-output')}
```

Comment:
Could `fleet-default-output` move to a constant?

Reply:
It should indeed! It's somewhere in the server part, I'll pull it out.